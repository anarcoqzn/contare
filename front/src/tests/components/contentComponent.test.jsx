import ContentComponent from '../../components/contentComponent/contentComponent';

it("should render correctly", () =>{
    const wrapper = shallow(
        <ContentComponent />
    );
    expect(wrapper).toMatchSnapshot();
});

