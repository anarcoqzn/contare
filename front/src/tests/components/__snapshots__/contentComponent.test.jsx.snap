// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="content"
>
  <img
    alt="background"
    src="planoDeFundo.jpg"
  />
  <div
    className="conteudo"
  />
</div>
`;
